src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,80 @@
  */
 
 public class Plane {
+    private String manufacturer;
+    private String model;
+    private int boardNumber;
+    private int passengerCapacity;
+    private int yearOfManufacturing;
 
+    private Plane(PlaneBuilder builder) {
+        this.manufacturer = builder.manufacturer;
+        this.model = builder.model;
+        this.boardNumber = builder.boardNumber;
+        this.passengerCapacity = builder.passengerCapacity;
+        this.yearOfManufacturing = builder.yearOfManufacturing;
+
+    }
+
+    public String getManufacturer() {
+        return manufacturer;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public int getBoardNumber() {
+        return boardNumber;
+    }
+
+    public int getPassengerCapacity() {
+        return passengerCapacity;
+    }
+
+    public int getYearOfManufacturing() {
+        return yearOfManufacturing;
+    }
+
+    public static class PlaneBuilder {
+        private String manufacturer;
+        private String model;
+        private int boardNumber;
+        private int passengerCapacity;
+        private int yearOfManufacturing;
+
+        public PlaneBuilder setManufacturer(String manufacturer) {
+            this.manufacturer = manufacturer;
+
```

Comment:
please remove this blank line in all setters

Reply:
done